components/si-labs/CP2105.stanza

```diff
@@ -99,10 +105,11 @@ public pcb-module module :
   pin gnd
   
   public inst xcvr : ocdb/si-labs/CP2105/component
+  ; connected as bus-powered device
   net (usb-2.vbus.vdd xcvr.VBUS xcvr.REGIN)
   cap-strap(xcvr.REGIN, gnd, 1.0e-6)
   net (usb-2.vbus.gnd xcvr.GND xcvr.EPAD gnd)
-  net (xcvr.VIO vio)
+  net (xcvr.VIO xcvr.VDD vio) ; connect VDD and VIO together
```

Comment:
@bhusang I don't think we want to hard-wire vio to vdd - the separate io voltage is a common use case (including being used on ble-mote).

You can just expose `vdd` and `vio` and let the user manage it

Reply:
@d-haldane  That's a better solution but I was thinking to preserve the connectivity that this component was already using. If that's not the case, I will separate vdd out.

Reply:
Better to make sure the use case is supported - port interfaces are pretty easy to update